src/module-elasticsuite-tracker/i18n/de_DE.csv

```diff
@@ -36,3 +36,7 @@
 "Bot handling","Handhabung von Robotern"
 "Filter bot hits","Filtern Sie Robotertreffer"
 "If enabled, it will prevent known IA or search engines crawler bots generated tracker hits from being recorded to prevent your behavioral data from being polluted by sometimes totally incoherent pagination or results count data.","Wenn diese Option aktiviert ist, wird verhindert, dass bekannte IA- oder von Suchmaschinen-Crawler-Bots generierte Tracker-Treffer aufgezeichnet werden, um zu verhindern, dass Ihre Verhaltensdaten durch manchmal völlig inkohärente Paginierungs- oder Ergebniszählungsdaten verunreinigt werden."
+"The tracker events seem to be frequently indexed as expected.","Die Tracker-Ereignisse scheinen regelmäßig wie erwartet indexiert zu werden."
+"There are currently %1 tracker events created more than %2 hours ago in the events queue table.","Aktuell wurden %1 Tracker-Ereignisse vor mehr als %2 Stunden in der Ereigniswarteschlange erstellt."
```

Comment:
Replace translation by 
"Derzeit befinden sich %1 Tracker-Ereignisse in der Ereigniswarteschlangentabelle, die vor mehr als %2 Stunden erstellt wurden."